### PR TITLE
Fix BFME mainmenu

### PR DESCRIPTION
**Pull request by feliwir**

### CODE REVIEW

File: src/OpenSage.Game/Gui/Apt/ActionScript/VM.cs [hunk 1]
```diff
@@ -85,7 +91,7 @@ public Value Execute(Function func, Value[] args, ObjectContext scope)
                 instr = stream.GetInstruction();
             }
 
-            return Value.Undefined();
+            return null;
```
Comment (tgjones) [inline src/OpenSage.Game/Gui/Apt/ActionScript/VM.cs]: Why return null here? What does this mean?

Comment (feliwir) [reply src/OpenSage.Game/Gui/Apt/ActionScript/VM.cs]: Functions can return a value when using the `Return` Instruction. Before i used the `Value.Undefined` to signal that there is no return code. But it seems like `Value.Undefined` is a valid return, so i changed this to be null. See this change here: https://github.com/OpenSAGE/OpenSAGE/pull/257/files#diff-af6541dbbcfe678870455d19e4e930b6R32

Comment (tgjones) [reply src/OpenSage.Game/Gui/Apt/ActionScript/VM.cs]: Right, makes sense.

**Comment by tgjones** [comment]
Thanks for the fix!

**Pull request merged**